_includes/configuration/inline.md

```diff
@@ -3,11 +3,9 @@
 
 {{site.notonmobile}}
 
-The **inline** option allows you to specify whether {{site.productname}} should work in inline mode.
+The **inline** option allows you to specify whether {{site.productname}} should initialize in inline mode.
```

Comment:
To me, "initialise" implies that it can be changed later, which isn't true unless you destroy and re-init the editor?

Reply:
I'd disagree here, initialize just means how it's started up, but I do agree we could probably use another word here to make it clearer. Maybe something as simple as `run`?

Reply:
changed to "run"